idb/postgres/internal/writer/writer_test.go

```diff
@@ -1394,9 +1394,10 @@ func TestWriterAddBlockInnerTxnsAssetCreate(t *testing.T) {
 	require.NotContains(t, txns[3].txn, "itx", "The inner transactions should be pruned.")
 
 	// Verify correct App and Asset IDs
-	require.Equal(t, 1, txns[0].asset, "intra == 0 -> ApplicationID = 1")
+	require.Equal(t, 1001, txns[0].asset, "intra == 0 -> ApplicationID = 1")
+	// TODO: How is this one allowed to be <1000 ?
 	require.Equal(t, 789, txns[4].asset, "intra == 4 -> ApplicationID = 789")
```

Comment:
@bbroder-algo this is one of our regression tests that calls into the go-algorand eval function. Somehow it has an appID < 1000

For additional context, I have no idea what this test is doing, I'm just adding 1000 to all the failing assertions.